ci/requirements-py3.13.yml

```diff
@@ -0,0 +1,28 @@
+name: test_env
+channels:
+    - defaults
+    - conda-forge
+dependencies:
+    - coveralls
+    - cython
+    - ephem
+    - h5py
+    - numba
+    - numpy >= 1.17.3
+    - pandas >= 1.3.0
+    - pip
+    - pytest
+    - pytest-cov
+    - pytest-mock
+    - requests-mock
+    - pytest-timeout
+    - pytest-rerunfailures
+    - conda-forge::pytest-remotedata  # version in default channel is old
+    - python=3.12
```

Comment:
This dependency looks kinda sus. Is it really necessary to pin python version in this file?

Reply:
I agreed that this seems strange

Reply:
```suggestion
    - python=3.13
```